clang/test/CIR/CodeGen/vector-ext.cpp

```diff
@@ -587,3 +587,118 @@ void foo11() {
 // OGCG: %[[TMP_B:.*]] = load <4 x i32>, ptr %[[VEC_B]], align 16
 // OGCG: %[[XOR:.*]] = xor <4 x i32> %[[TMP_A]], %[[TMP_B]]
 // OGCG: store <4 x i32> %[[XOR]], ptr {{.*}}, align 16
+
+void foo12() {
```

Comment:
Can you add test cases for unsigned elements and floating-point?

Reply:
Unsinged elements will require upstreaming Bitcase support, I will upstream it, then update the test files